### ISSUE TITLE
Any chance of packaging on melpa

### ISSUE THREAD

**Issue by olymk2** [kuanyui/browser-f5.el#1]
Awesome little package be nice if it could be installed via melpa.

**Comment by kuanyui** [comment]
Glad to hear someone is using this.

I'm a little hesitate if I should throw a PR onto MELPA currently, because I don't sure is this method reliable enough on each window manager. 

You can take a look of source code, I search window by checking if the window's title name is **suffixed** with browser's name. (e.g. Regexp: `.Mozilla Firefox$`)

However this is customizable from window manager, user can make `Mozilla Firefox` as the prefix of window's title text.

**Comment by olymk2** [comment]
Good point the only thing I can think of is could you use the process name and then use xdotools search by pid to get the window id ?

not sure if thats any more reliable but seems it would be less likely to change ? 

Alternatively make the window title variable changeable by users, on my system it picks up firefox and chrome.
